app/assets/javascripts/models/beta/story.js

```diff
@@ -75,6 +75,11 @@ export function update(story, projectId) {
     }));
 };
 
+export function deleteStory(storyId, projectId) {
```

Comment:
why are we using an arrow function syntax bellow and here we are using normal syntax? perhaps if we choose one of the two styles it would be better :)